client/state/i18n/middleware.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @format
+ */
+
+/**
+ * Internal dependencies
+ */
+
+import { LOCALE_SET } from 'state/action-types';
+
+function updateMoment( action ) {
+	if ( action && action.type === LOCALE_SET && action.localeSlug !== 'en' ) {
+		// load moment and update the locale
+		import( 'moment' ).then( moment => {
+			import( /* webpackInclude: /\.js$/ */
```

Comment:
Nice trick! The `webpackInclude` option tells webpack to bundle all `*.js` files in the directory?

Can we also instruct Webpack to give these chunks reasonable names? It's all numbers now, `moment-locale-cs` would be a better name.